segysak/segy/_segy_loader.py

```diff
@@ -35,6 +35,8 @@
     petrel_2d=dict(cdp=21, cdpx=73, cdpy=77),
 )
 
+UNRECOGNIZED_OFFSET = 'Unknown Enum'
+
```

Comment:
Defining this module level constant is potentially YAGNI (you aren't going to need it) as it's only used in a single place in this module. My rationale for doing things this way was to semi-future-proof things a little bit. We're relying on `segyio`'s error message for a bad byte offset value to always be `Unkown Enum`, which is a fragile assumption and could break in newer versions of `segyio`